ext/panko_serializer/panko_serializer.c

```diff
@@ -131,15 +135,16 @@ VALUE serialize_subjects(VALUE key,
                          VALUE str_writer,
                          SerializationDescriptor descriptor,
                          VALUE context) {
+  long i;
+
   rb_funcall(str_writer, push_array_id, 1, key);
```

Comment:
You could use rb_ary_push instead.

Reply:
this code is calling `push_array` on `Oj::StringWriter` , how `rb_ary_push` fits in?

Reply:
my bad, I should have read more carefully.

Reply:
👍